doc/source/building/working_with_images/iso_to_usb_stick_file_based_deployment.rst

```diff
@@ -0,0 +1,63 @@
+.. _iso_as_file_to_usb_stick:
+
+Deploy ISO Image as file on a FAT32 formated USB Stick
+======================================================
+
+.. sidebar:: Abstract
+
+   This page provides further information for handling
+   ISO images built with KIWI and references the following
+   articles:
+
+   * :ref:`hybrid_iso`
+
+In KIWI all generated ISO images are created to be hybrid. This means,
+the image can be used as a CD/DVD or as a disk. The deployment of such
+an image onto a disk like an USB stick normally destroys all existing
+data on this device. Most USB sticks are pre-formatted with a FAT32
+Windows File System and to keep the existing data on the stick untouched
+a different deployment needs to be used.
+
+The following deployment process just copies the ISO image as an
+additional file to the USB stick and makes the USB stick bootable.
+The ability to boot from the stick is configured through a grub
```

Comment:
grub -> GRUB

I don't know if we should explicitly mention the version (probably GRUB 2)? Just an idea...